friends/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "friends",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "axios": "^0.18.0",
+    "react": "^16.6.0",
+    "react-dom": "^16.6.0",
+    "react-loading": "^2.0.3",
+    "react-redux": "^5.0.7",
+    "react-router": "^4.3.1",
+    "react-router-dom": "^4.3.1",
+    "react-scripts": "2.0.5",
+    "redux": "^4.0.1",
+    "redux-devtools-extension": "^2.13.5",
```

Comment:
very cool that you figured out how to get redux devtools to work as an npm package. Good work. Next time, however, install it as a dev-dependency since you don't want the package to be part of your deployed build. 🚀